### PR TITLE
Add quantum packages and discipline

### PR DESCRIPTION
**Pull request by TheKharleeci**
## Proposed changes

[preview page](https://csc-guide-preview.rahtiapp.fi/origin/ft-add-quantum-discipline/)
- add quantum discipline
- add qiskit-iqm and cirq-iqm to apps
- move pennylane, qiskit and cirq-iqm to quantum discipline

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes all tests.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/ft-add-quantum-discipline/).

### CODE REVIEW

File: docs/apps/cirq-on-iqm.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+tags:
+  - Free
+---
+
+# Cirq-on-iqm
+
+Cirq on IQM is an open-source cirq adapter for IQM quantum computers. It is installed as helmi_cirq on LUMI. It is used for running quantum circuits on Helmi.
```
Comment (JMuff22) [inline docs/apps/cirq-on-iqm.md:8]:
```suggestion
Cirq on IQM is an open-source cirq adapter for IQM quantum computers. It is installed as `helmi_cirq` on LUMI. It is used for running quantum circuits on [Helmi](../computing/quantum-computing/helmi/running-on-helmi.md).
```

File: docs/apps/qiskit-on-iqm.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+The module includes python packages that are often used with qiskit, such as qiskit-experiments, matplotlib, numpy, and jupyterlab.
+
+## License
+
+qiskit-on-iqm is licensed under
+[Apache License 2.0](https://github.com/iqm-finland/qiskit-on-iqm/blob/main/LICENSE).
+
+## Usage
+
+To use helmi_qiskit on LUMI, initialize it with:
```
Comment (JMuff22) [inline docs/apps/qiskit-on-iqm.md:32]:
```suggestion
To use `helmi_qiskit` on LUMI, initialize it with:
```

File: docs/apps/qiskit-on-iqm.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+---
+tags:
+  - Free
+---
+
+# Qiskit-on-iqm
+
+Qiskit on IQM is an open-source qiskit adapter for IQM quantum computers. It is installed as helmi_qiskit on LUMI. It is used for running quantum circuits on Helmi.
```
Comment (JMuff22) [inline docs/apps/qiskit-on-iqm.md:8]:
```suggestion
Qiskit on IQM is an open-source qiskit adapter for IQM quantum computers. It is installed as `helmi_qiskit` on LUMI. It is used for running quantum circuits on [Helmi](../computing/quantum-computing/helmi/running-on-helmi.md).
```

File: docs/apps/cirq-on-iqm.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+module use /appl/local/quantum/modulefiles
+```
+
+and 
+
+```bash
+module load helmi_cirq
+```
+
+This command will also show how to load helmi_cirq:
```
Comment (JMuff22) [inline docs/apps/cirq-on-iqm.md:44]:
```suggestion
This command will also show how to load `helmi_cirq`:
```

File: docs/apps/cirq-on-iqm.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+To use helmi_cirq on LUMI, initialize it with:
+
+```bash
+module use /appl/local/quantum/modulefiles
+```
+
+and 
+
+```bash
+module load helmi_cirq
```
Comment (JMuff22) [inline docs/apps/cirq-on-iqm.md:41]:
```suggestion
module load `helmi_cirq`
```

File: docs/apps/cirq-on-iqm.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  - Free
+---
+
+# Cirq-on-iqm
+
+Cirq on IQM is an open-source cirq adapter for IQM quantum computers. It is installed as helmi_cirq on LUMI. It is used for running quantum circuits on Helmi.
+
+## Available
+
+Currently supported qiskit-on-iqm versions:
```
Comment (JMuff22) [inline docs/apps/cirq-on-iqm.md:12]:
```suggestion
Currently supported [cirq-on-iqm](https://iqm-finland.github.io/cirq-on-iqm/) versions:
```

File: docs/apps/qiskit-on-iqm.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+## Available
+
+Currently supported qiskit-on-iqm versions:
```
Comment (JMuff22) [inline docs/apps/qiskit-on-iqm.md:12]:
```suggestion
Currently supported [qiskit-on-iqm](https://iqm-finland.github.io/qiskit-on-iqm/) versions:
```

**Review by JMuff22 — APPROVED** [review]
LGTM! Some minor formatting changes.

**Review by joonas-somero — APPROVED** [review]
Another one of these weird and unintuitive quantum pull requests, or _QPRs_ if you will.

**Comment by joonas-somero** [comment]
> > Another one of these weird and unintuitive quantum pull requests, or _QPRs_ if you will.
> 
> Is it quite confusing? if so the my work is not done

I was joking. Or at least trying to. :smile:

**Pull request merged**